src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -341,13 +341,13 @@
     <value>Symbols should be compared for equality, not identity. Use an overload accepting an 'IEqualityComparer' and pass 'SymbolEqualityComparer'.</value>
   </data>
   <data name="CompareSymbolsCorrectlyMessage" xml:space="preserve">
-    <value>Compare symbols correctly</value>
+    <value>Use 'SymbolEqualityComparer' when comparing symbols</value>
   </data>
   <data name="CompareSymbolsCorrectlyTitle" xml:space="preserve">
-    <value>Compare symbols correctly</value>
+    <value>Symbols should be compared for equality</value>
```

Comment:
If I understand correctly, this is misleading. A code violating this rule is already comparing for equality. e,g `symbolA.Equals(symbolB)`, and this is completely equivalent to `symbolA.Equals(symbolB, SymbolEqualityComparer.Default)`.

The thing is mostly about the "intent" when there are nullable differences.